command/kv_patch.go

```diff
@@ -0,0 +1,196 @@
+package command
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var _ cli.Command = (*KVPatchCommand)(nil)
+var _ cli.CommandAutocomplete = (*KVPatchCommand)(nil)
+
+type KVPatchCommand struct {
+	*BaseCommand
+
+	flagCAS   int
```

Comment:
I don't think this is getting used anywhere. I wonder if we should allow a cas flag that verifies the retrieved version equals that flag

Reply:
We could, although in that case you might as well have just modified the output of that initial read. Could defer until someone asks.

Reply:
True, i'm fine with that